app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<section class='container'>
+  <% @recipes.each do |recipe| %>
+    <div class='card'>
+      <div class='subdiv'>
+        <a href="<%= recipe_path(recipe.id)%>">
+          <h2><%= recipe.name %></h2>
+        </a>
+        <%= button_to 'Delete', recipe_delete_path(recipe.id), method: :delete %>
+      </div>
+      <div class='subdiv'>
+        <p><%= recipe.description %></p>
+      </div>
+    </div>
```

Comment:
- [ ] @asdt560 and @d4nQw3rty good work on your views however it's best practice to use partial views for your recipes, kindly add it as you did for the foods.